learning/tour-of-beam/frontend/lib/components/login/login_content.dart

```diff
@@ -0,0 +1,140 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:playground_components/playground_components.dart';
+
+import '../../constants/sizes.dart';
+import '../../generated/assets.gen.dart';
+
+class LoginContent extends StatelessWidget {
+  const LoginContent();
+
+  @override
+  Widget build(BuildContext context) {
+    return _Body(
+      child: Column(
+        children: [
+          Text(
+            'ui.signIn',
+            style: Theme.of(context).textTheme.titleLarge,
+          ).tr(),
+          const SizedBox(height: BeamSizes.size10),
+          const Text(
+            'dialogs.signInIf',
+            textAlign: TextAlign.center,
+          ).tr(),
+          const _Divider(),
+          const _BrandedLoginButtons(),
+        ],
+      ),
+    );
+  }
+}
+
+class _Body extends StatelessWidget {
```

Comment:
Why this widget exists? It is not reused anywhere, so we can just put this login into LofinContext.build method

Reply:
I extract widgets to also improve readability.

1. Having this widget in the `build` method would decrease readability.
2. I associate `_Background` with something behind a widget that can have a color.